items/items-perm.md

```diff
@@ -196,26 +217,34 @@ To decrease the propagation level (whatever the level), you do not need any spec
 ### Granted permissions table (permissions_granted)
 
 The `permissions_granted` table express the raw permissions given to a group on an item.
-One permission entry matches exactly one group, one item, one *source group* and one *origin*. The *source group* is the group from which the destination group has received the permission. Only the managers of this *source group* are able to revoke or modify this permission. The *origin* is the process which has allocated this permission (group permission, unlocking, ...). There may be several permissions for a same *(group, item)*, e.g., multiple permissions to the same task given to an end-user by several groups he is member of.
+One permission entry matches exactly one `group`, one `item`, one `source_group` and one `origin`.
+
+The `source_group` is the group from which the destination group has received the permission.
+Only the managers of this `source_group` are able to revoke or modify this permission.
+
+The `origin` is the process which has allocated this permission (group permission, unlocking, ...).
+There may be several permissions for a same *(`group`, `item`)*, e.g.,
+multiple permissions to the same task given to an end-user by several groups he is a member of.
```

Comment:
repeat what has been said multiple time (but that's ok if you think that's better)